deployments/liqo/templates/liqo-remote-peering-rbac.yaml

```diff
@@ -12,6 +12,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ include "liqo.prefixedName" $peeringBasic }}
+  labels:
+    {{- include "liqo.labels" $peeringBasic | nindent 4 }}
+    auth.liqo.io/remote-peering-permissions: "basic"
```

Comment:
Since these labels are used in code, could we add a comment that reminds us to change the label in both places if we ever need to change them?